python/lsst/faro/measurement/example_VisitMeasurementTasks.py

```diff
@@ -31,8 +31,8 @@ class StarFracTask(Task):
     ConfigClass = Config
     _DefaultName = "starFracTask"
 
-    def run(self, catalog, metric_name, vIds):
-        self.log.info("Measuring %s", metric_name)
+    def run(self, catalog, metricName, vIds):
```

Comment:
I think the order would be metricName, catalog, vIds for consistency with other run methods